examples/proxy/index.js

```diff
@@ -0,0 +1,47 @@
+const { Pool, Client } = require('../../')
+const http = require('http')
+const proxy = require('./proxy')
+
+const client = new Pool('http://localhost:4001', {
+  connections: 256,
+  pipelining: 1
+})
+
+Promise.all([
+  new Promise(resolve => {
+    // Proxy
+    http.createServer(async (req, res) => {
```

Comment:
let's not use async here. `writeHead()` could potentially throw and we'll be in a unhandledRejection situation.